.buildkite/ftr_configs.yml

```diff
@@ -79,24 +79,13 @@ disabled:
   - x-pack/test_serverless/functional/config.base.ts
   - x-pack/test_serverless/shared/config.base.ts
 
-  # Serverless configs, currently only for manual tests runs, CI integration planned
-  - x-pack/test_serverless/api_integration/test_suites/common/config.ts
-  - x-pack/test_serverless/api_integration/test_suites/observability/config.ts
+  # Serverless feature flag config files
   - x-pack/test_serverless/api_integration/test_suites/observability/config.feature_flags.ts
```

Comment:
Is there a reason why we don't include the serverless feature flag confgurations?

Reply:
On second pass it looks like I can add back the observability api integration feature flag configurations.  The other suites have no tests, and therefore no junit report is produced which causes CI to exit with a failure.  The exit with a failure seems a little odd, but figured it was more suitable for a different PR.

Reply:
I pushed https://github.com/elastic/kibana/pull/159607/commits/49d67ac4059d57ef050509cd261e1fb4c48d5c38

Reply:
I see. We created the other configs as placeholders so folks know where to add their feature flag tests. I didn't expect them to cause issues. But it's fine to exclude them for now.